pkg/plugins/manager/signature/manifest.go

```diff
@@ -194,6 +200,9 @@ func Calculate(ctx context.Context, mlog log.Logger, src plugins.PluginSource, p
 	// Track files missing from the manifest
 	var unsignedFiles []string
 	for _, f := range plugin.FS.Files() {
+		// Ensure slashes are used, because MANIFEST.txt always uses slashes regardless of the filesystem
```

Comment:
Do you happen to know which plugin had this?

This old behaviour had `filepath.ToSlash`, which suggests all paths in MANIFEST.txt should have always use slashes rather than backslashes.

https://github.com/grafana/grafana/pull/63734/files#diff-a5c25122620a6907c043637c2464cc6803cfe79ca9c455cd800a8a7f661e924fL283

This is also what sign-plugin does:
https://github.com/grafana/plugin-tools/blob/7d9b188ebbdfe17567832994b52dc6c56ab2dac9/packages/sign-plugin/src/utils/manifest.ts#L66

And I believe the same is also for grafana-toolkit:
https://github.com/grafana/grafana/blob/2031997994af807d7b3ee2bd01d04965e8a44e2a/packages/grafana-toolkit/src/plugins/manifest.ts#L9

@wbrowne any ideas?

Reply:
Sadly already trashed the machine but I think that was in grafana-image-renderer.

Reply:
I just tried with the Renderer plugin build for WIndows and it uses POSIX `/`. I don't think I've ever come across one that does otherwise

Reply:
I have also tried downloading the latest clickhouse plugin for windows (backend datasource) and the latest clock plugin (panel), both of them had their paths with slashes:

https://grafana.com/grafana/plugins/grafana-clickhouse-datasource/?tab=installation
```
  "files": {
    "CHANGELOG.md": "f79f45377225e9f45bcd4532d91a2ee014b1beabaad4c861f3c2ccc32e5f6f9a",
    "LICENSE": "cdd15e614b50e88443fe574ad56bde5ba697d958a45376431638eea816e3bfc3",
    "README.md": "da1ece921b4bc2c1de0c0e16f86559f740c53a4310e8c73aceb584f21b0079ba",
    "dashboards/cluster-analysis.json": "e5be94806dcef74e8948230234845e1adebce2f69e50abae0cb73550ac07ff3d",
    "dashboards/data-analysis.json": "cfc36e74161ee07691fc8fd96873c6f858afab08a24400104e6a59de608a3b3d",
    "dashboards/query-analysis.json": "47b28a56a00f0b418dbee317eb43748cb4270b0c2f1bfdee5915bfecb2c05a14",
    "go_plugin_build_manifest": "bb0047c559ce6b1866a6cc67d736362f29bbcdcb865948aaa031b08cea711463",
    "gpx_clickhouse_windows_amd64.exe": "66710413dbcdd447198bbcaf8583231211055f1a5d8c0721812de16123f17e3e",
    "img/logo.png": "57b548fd842706f2845514f661aed5d4b4aeefeb6b6a891bac7cc0392b3ac060",
    "img/logo.svg": "1defc6f7e585c67657bcfd8fddc599ee7dfa82f8674413f49fa274c2cd453ec6",
    "module.js": "6730a5361e10f5b84024337b759482fc1f3720659883dddda6e45bfab036bb96",
    "module.js.LICENSE.txt": "4de3dd152c804e513c23fbd7d770c3ee1a035af760fb00c3063c66979be3afa4",
    "module.js.map": "ca1976140b781b0924f67dcacddd9fa988d92a62341bfd58c50ec4add012e52a",
    "plugin.json": "6e1ae35a5342b9f7850e92730824c67d49295fdc3adf2b9ee1ff9cea20a4b15b"
  }
```

https://grafana.com/grafana/plugins/grafana-clock-panel/?tab=installation
```
  "files": {
    "CHANGELOG.md": "d9183b9e669b1ee94da93626a0f99d874b07c71adbe5b8a8213815f9e7c76f7f",
    "LICENSE": "b46400c1ab5630edeb4e43b240a027d203375ebe5a9efb7de88c604c5c5cace0",
    "README.md": "265b2b3c4f46881ff656089aa2561f98ae672c6045f7db73e667853d2ff11960",
    "img/clock.svg": "d27cbdb85432fe1ad96eeec6758c2c4a0b189b306ba2040519aec09938889fbc",
    "img/countdown1.png": "ce51c235273667dde4a8fedc36ea70c6bfa7670b247514a2ae1654bd872548aa",
    "img/screenshot-clock-options.png": "55f428cc45f65e53a118db2b969353cd01e0d2f879848b1df503750ed27007cf",
    "img/screenshot-clocks.png": "4dcf3c3d303e63239689df853b80ef79557eaf5496679400f362e68f57f60577",
    "img/screenshot-showcase.png": "43c6520f1efa3343d839b9462a22ce962d2b041f2d5d9c743a8603caa8392a71",
    "module.js": "6a3dc5f875470737568b3017681d0a27fba119da93b2a07782add16bfe808e17",
    "module.js.LICENSE.txt": "71f2699fd278800e3b685a3232025864b829081d5c892e6a3cb66b3e3fdf0cbf",
    "module.js.map": "2aff06618472afb7162533c81202f0cfad6984d1a511ae811a83e03b62121db1",
    "plugin.json": "fc2d51bfefed5488e131a9fe67d1dfa9b3ad9dc7811703a49e210d03350da112"
  }
```

Reply:
OK, perhaps this was also created by this bug? But if you can install that plugin locally on Windows, then I think we're good. Thanks <3